### PR TITLE
readme fixes

### PR DESCRIPTION
**Pull request by dougsillars**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,17 +25,22 @@ jobs:
         uses: bump-sh/github-action@0.2
         with:
           id: <BUMP_DOC_ID>
-          token: <BUMP_DOC_TOKEN>
+          token: ${{secrets.BUMP_TOKEN}}
```
Comment (antlio) [inline README.md:28]: what do you think about keeping the example using the following `<BUMP_DOC_TOKEN>`?

File: README.md [hunk 2]
```diff
@@ -25,17 +25,22 @@ jobs:
...
           file: doc/api-documentation.yml
 ```
 
 Important: [actions/checkout](https://github.com/actions/checkout) has to be called **before this action**.
 
+
 ## Inputs
 
 * `id` (required): Documentation id. Can be found in the documentation settings on https://bump.sh
```
Comment (antlio) [inline README.md:37]: shouldn't we update `id` -> `doc` here too?

**Review by antlio — CHANGES_REQUESTED** [review]
thanks for your contribution, lgtm!

File: README.md (hunk 2 above)
Comment (dougsillars) [reply README.md:37]: its in the 2nd commit.

**Review by antlio — APPROVED** [review]

**Pull request merged**